tools/sim/lib/simulated_car.py

```diff
@@ -107,6 +107,7 @@ def send_panda_state(self, simulator_state):
       'controlsAllowed': True,
       'safetyModel': 'hondaNidec',
       'alternativeExperience': self.sm["carParams"].alternativeExperience
+      'safetyParam': 16
```

Comment:
just beat you :) here https://github.com/commaai/openpilot/pull/30738

always better to use the actual param rather than hardcoding